packages/docmanager-extension/src/index.ts

```diff
@@ -473,7 +473,9 @@ function addCommands(
       }
 
       return docManager.services.contents.getDownloadUrl(path).then(url => {
-        window.open(url, '_blank');
+        const opened = window.open('', '_blank', 'noopener');
```

Comment:
The second and third arguments here should be removed. `_blank` is the default second argument anyway and `noopener` is handled by setting it to `null` in your next line anyway  :)